dlib/test/rand.cpp

```diff
@@ -432,7 +432,7 @@ namespace
         print_spinner();
         dlib::rand rnd(0);
         
-        const size_t N = 1024*1024*128;
+        const size_t N = 1024*1024*16;
```

Comment:
These changes don't seem to be needed in this PR

Reply:
Yes that's true. I knew you would spot that. I realised that test takes ages... Using 1024*1024*16 was sufficient. I will undo that commit. Maybe that can be another tinee tiny PR.